frontends/election-manager/src/components/smartcard_modal/status_message.tsx

```diff
@@ -74,34 +78,36 @@ export function StatusMessage({
   }
 
   if (action === 'Program' && programmedUser) {
+    // No need to include the programmed user role in these messages since it's so prominent
+    // elsewhere on the screen
```

Comment:
I only noted this because other messages in this component, error and in-progress, do include user roles. And I could see someone thinking we just forgot to include them here when it was actually a conscious choice

Reply:
Despite the above, thought some more about this one and agree that this isn't necessary! https://github.com/votingworks/vxsuite/pull/2267/commits/0769554e8bb31e5a44876b96a1ff5955abe77c45

It may even lose relevance as we potentially shorten other status messages to not include user role either